brickstrap.sh

```diff
@@ -40,35 +40,43 @@ function br_script_path()
     echo -n "$SCRIPT_PATH"
 }
 
+. "$(br_script_path)/brickstrap-argv.sh"
 . "$(br_script_path)/brickstrap-components.sh"
 . "$(br_script_path)/brickstrap-utils.sh"
 
 # Commandline options. This defines the usage page
 BRP_USAGE=$(cat <<'EOF'
-Usage: brickstrap -p <project> [-c <component>] [-d <dir>] <command>
+Usage: brickstrap -p <project> [-c <component>] [-d <dest>] <command>
 
 Options
 -------
 -c <component> Select components from the project directory.
                Multiple components may be selected by specifying multiple '-c'
                options; at least one component must be specified.
--d <dir>       The name of the directory for the rootfs.
-               Note: This is also used for the .tar and .img file names.
+-d <dest>      Destination directory in which brickstrap should store output.
 -p <project>   Directory which contains the brickstrap configuration (project).
                This option is required and should occur exactly once.
                Values are either a path to the project directory or the name of
                an example project shipped with brickstrap by default.
--Q <qemu>      Optional: override which QEMU binary to use for emulation of
+-I <image>     Name of the disk image file to generate (without type suffix).
```

Comment:
I'd like to leave type suffixes to the driver. When generating multiple variations of the same rootfs image (multiple drivers) the image name should reflect which driver generated it and ideally the image suffix should reflect something about the type of disk image (e.g. `.vmdk` or `.mbr`/`.img`) generated.

Injecting that information probably breaks expectations if you let the user pick suffixes as well because then they might quite reasonably think they control the actual output name instead of just a 'template' value which information is tacked onto.

Users that are really concerned about this can write a custom deployment script that fixes this with `mv` commands.